### PR TITLE
docs: Fix typos and formatting

### PR DESCRIPTION
**Pull request by jodh-intel**
Correct spelling mistakes and formatting issues.

Fixes: #571.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

File: kata-deploy/README.md [hunk 1]
```diff
@@ -10,15 +10,15 @@
     * [Remove Kata from the Kubernetes cluster](#remove-kata-from-the-kubernetes-cluster)
 * [`kata-deploy` details](#kata-deploy-details)
     * [Dockerfile](#dockerfile)
-    * [Daemonsets and RBAC](#daemonsets-and-rbac)
+    * [DaemonSets and RBAC](#DaemonSets-and-rbac)
```
Comment (grahamwhaley) [inline kata-deploy/README.md]: tocs/label refs don't do upper case do they? I suspect this was the victim of a global search/replace?

Comment (egernst) [reply kata-deploy/README.md]: correct - agreed

Comment (jodh-intel) [reply kata-deploy/README.md]: Good catch - fixed!

**Comment by jodh-intel** [comment]
@egernst - In the custom dictionary (which you'll be able to see later today hopefully), I've been very careful to use "correct"/official names. That particular term should use camel-case fwics:

- https://kubernetes.io/docs/concepts/workloads/controllers/daemonset

Once all my "typo + formatting" PRs have landed, all Kata docs will consistently use that approach. But I could add a custom rule if we need to bend the rules?

**Pull request merged**